source/install/desktop-app-install.rst

```diff
@@ -202,6 +200,16 @@ The following additional documentation resources are also available for the Matt
 Troubleshooting your Desktop App installation
 ----------------------------------------------
 
+Where is configuation stored locally?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The location of the Mattermost desktop app configuration file depends on the platform where you're running Mattermost (and, in the case of macOS, how you've chosen to install the app):
+
+- Windows: ``Users\<username>\AppData\Roaming\Mattermost``
+- macOS standalone download: ``/Users/<username>/Library/Application Support/Mattermost``
+- macOS App Store & TestFlight: ``/Users/<username>/Library/Containers/Mattermost/Data/Library/Application Support/Mattermost``
```

Comment:
I think we could/should drop "& TestFlight". I say this as its an Apple delivery mechanism, which is basically the second of the two path variants.